static/index.js

```diff
@@ -0,0 +1,82 @@
+let reviews = [];
+
+let sum = 0;
+
+if(localStorage.getItem("reviews") !== null) reviews = JSON.parse(localStorage.getItem("reviews"));
+
+
+
+const updateAverage = () =>{
+    document.querySelector("#averageRating").innerText = `${Math.round(sum / reviews.length * 10) / 10}/5`;
+}
+
+const createItem = (item) => {
+    let reviewItem = document.createElement("div");
+    let reviewText = document.createElement("p");
+    reviewText.innerText = item.review;
+    
+    let ratingText = document.createElement("p");
+    ratingText.innerText = `${item.rating} / 5`;
+
+    let deleteButton = document.createElement("button");
+    deleteButton.innerText = "Delete";
+    deleteButton.id = `item-${item.id}`;
+    
+    updateAverage();
+
+    deleteButton.addEventListener("click", (e)=>{
+        for(let rev in reviews){
+            console.log(`${e.target.id.split('-')[1] } VS..... ${reviews[rev].id}`)
+
+            if(e.target.id.split('-')[1] == reviews[rev].id) {
+                sum -= reviews[rev].rating;
+                console.log("brisam", reviews[rev]);
+                reviews.splice(rev, 1);
+                break;
+            }
+            
+        }
+    
+        localStorage.setItem("reviews", JSON.stringify(reviews));
+
+        updateAverage();
+
+        const currentItem = e.target.parentElement;
+        currentItem.parentElement.removeChild(currentItem);
+
+
+        console.log(reviews);
+
+    })
+
+    reviewItem.append(reviewText, ratingText, deleteButton);
+
+    return reviewItem;
+}
+
+
+reviews.forEach(item => {
```

Comment:
If this was extracted to a function, it would be much more readable and it would be easier to maintain later.

Reply:
Fixed in [Improvements](https://github.com/infinum-academy/js-react-academy-2024-filiptodosov/pull/1/commits/79575e604ff61d5206a96a781ad054151af9c333)